src/main/kotlin/org/dash/mobile/explore/sync/process/DashDirectDataSource.kt

```diff
@@ -136,35 +157,23 @@ class DashDirectDataSource(private val devApi: Boolean, slackMessenger: SlackMes
                     logger.info("DashDirect.totalRows: ${responseData.totalRows}")
 
                     responseData.merchants.forEach { merchant ->
+                        merchant.locations.forEach { location ->
+                            val locationData = location.asJsonObject
 
-                        val merchantData = merchant.merchant
-                        if (merchantData.get("IsActive").asBoolean) {
```

Comment:
The `merchant` object doesn't have the `IsActive` field anymore, so we must assume all returned merchants are active.
`IsActive` is still present in the `location` object.